wurst/utils/UnitUtils.wurst

```diff
@@ -0,0 +1,12 @@
+package UnitUtils
+
+public function findNearestUnit(vec2 pos, real range, filterfunc filter) returns unit
+    GroupEnumUnitsInRange(ENUM_GROUP, pos.x, pos.y, range, filter)
+    unit nearest = null
+    var bestDist = REAL_MAX
+    for u from ENUM_GROUP
+        let distSq = pos.distanceToSq(u.getPos())
+        if distSq < bestDist
+            nearest = u
+            bestDist = distSq
+    return nearest
```

Comment:
Did you perhaps miss this

https://gyazo.com/0326ee81b1359bc4f7c0182db45a4118

Reply:
@Marsunpaisti I didn't, that method returns the unit in a closure which I didn't want (and is unnecessary)